FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -204,7 +204,7 @@ public static function useShopSearch()
     {
         $request = Shopware()->Front()->Request();
         $isCLIMode = $request === null;
-        if ($isCLIMode) {
+        if ($isCLIMode || !Shopware()->Container()->has('shop')) {
```

Comment:
Add a comment above the condition, mentioning when no shop is available, and why we need this check.